### PR TITLE
[SUCE-441] Various stylistic tweaks by Sujato

### PR DESCRIPTION
**Pull request by SamaneriVimala**

### CODE REVIEW

File: client/elements/suttaplex/sc-parallel-item.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
       .parallel-item-main-info-container {
         width: 100%;
-        max-width: 320px;
+        max-width: 360px;
```
Comment (TrebuhD) [inline client/elements/suttaplex/sc-parallel-item.html]: This should be tested on mobile devices in chrome devtools - particularly if it fits inside the smallest devices' viewports and doesn't blow up the layout horizontally.

File: client/elements/suttaplex/sc-parallel-list.html [hunk 2]
```diff
@@ -50,6 +47,7 @@
 
       .parallels-root-cell {
         text-align: center;
+        min-width:90px;
```
Comment (TrebuhD) [inline client/elements/suttaplex/sc-parallel-list.html]: missing space

File: client/styles/sc-text-styles.html [hunk 3]
```diff
@@ -368,7 +368,7 @@
 
       hr {
         width: 33%;
-        margin: 1.5rem auto
+        margin: 1em auto 0
```
Comment (TrebuhD) [inline client/styles/sc-text-styles.html]: missing semicolon

File: client/styles/sc-text-styles.html [hunk 4]
```diff
@@ -252,6 +246,12 @@
         color: var(--sc-secondary-text-color);
         @apply --sc-sans-font;
       }
+      
+      #text:lang(pli) p,
+      #text:lang(san) p {
+        word-wrap: break-word;
+        overflow-wrap: break-word
```
Comment (TrebuhD) [inline client/styles/sc-text-styles.html]: missing semicolon

File: client/elements/suttaplex/sc-parallel-item.html (hunk 1 above)
Comment (sujato) [reply client/elements/suttaplex/sc-parallel-item.html]: I have tested it and it seems fine.

**Pull request merged**